### ISSUE TITLE
[a11y] Cannot run reports

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1185]
**Describe the bug**

**This error appears over 70 times**

![image](https://user-images.githubusercontent.com/37256050/158607895-ad2541b9-f6e0-469d-9e7c-2e1210e88514.png)
This is the console error that repeats.

```
Unexpected Error

An unexpected error has occurred

Please reload the page and try again.
If this issue persists, please contact your IT support or if this is a Specify Cloud database, contact [support@specifysoftware.org](mailto:support@specifysoftware.org)

Error Message
TypeError: Cannot read properties of undefined (reading 'mappingPath')
at w (https://a11y.test.specifysystems.org/static/js/636.3db06bff2c62c14dfb33.bundle.js:2:78648) at c (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:51405) at E (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:26468)
```

**To Reproduce**
1. Run a report
2. See all 70+ errors

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge
